aspnetcore/migration/22-to-30.md

```diff
@@ -437,9 +437,7 @@ Apps and libraries targeting ASP.NET Core 2.1 or later should remove any direct
 
 As part of the work to [improve the ASP.NET Core shared framework](https://blogs.msdn.microsoft.com/webdev/2018/10/29/a-first-look-at-changes-coming-in-asp-net-core-3-0/), [Json.NET](https://www.newtonsoft.com/json/help/html/Introduction.htm) has been removed from the ASP.NET Core shared framework.
 
-The default for ASP.NET Core is now [System.Text.Json](/dotnet/api/system.text.json?view=netcore-3.0), which is new in .NET Core 3.0. Consider using `System.Text.Json` when possible. It's high-performance and doesn't require an additional library dependency. However, since `System.Text.Json` is new, it might currently be missing features that your app needs.
-
-Your app may require `Newtonsoft.Json` integration if it uses `Newtonsoft.Json`-specific feature such as JsonPatch or converters or if it [formats](xref:web-api/advanced/formatting) `Newtonsoft.Json`-specific types.
+The default for ASP.NET Core is now [System.Text.Json](/dotnet/api/system.text.json?view=netcore-3.0), which is new in .NET Core 3.0. Consider using `System.Text.Json` when possible. It's high-performance and doesn't require an additional library dependency. However, since `System.Text.Json` is new, it might currently be missing features that your app needs. For more information, see [How to migrate from Newtonsoft.Json to System.Text.Json](/dotnet/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to).
 
 To use Json.NET in an ASP.NET Core 3.0 SignalR project, see [Switch to Newtonsoft.Json](#switch-to-newtonsoftjson) in this document.
```

Comment:
We should consider updating uses of `Json.NET in this document. However, I have seen folks link to this heading in a bunch of places, so maybe there is a way to do it without disrupting those links.

Ideally, we would mention this in one place (maybe the heading) and then consistently use `Newtonsoft.Json` everywhere else in the docs: Newtonsoft.Json (a.k.a. Json.NET)

```suggestion
To use Newtonsoft.Json (a.k.a. Json.NET) in an ASP.NET Core 3.0 SignalR project, see [Switch to Newtonsoft.Json](#switch-to-newtonsoftjson) in this document.
```